### PR TITLE
Add support for Netgear WNR3500L v2

### PR DESCRIPTION
**Pull request by ffainelli**
Thanks for your contribution to OpenWrt!

To help keep the codebase consistent and readable,
and to help people review your contribution,
we ask you to follow the rules you find in the wiki at this link
https://openwrt.org/submitting-patches

Please remove this message before posting the pull request.

### CODE REVIEW

**Comment by ffainelli** [comment]
I will wait until the upstream patches settle so we can merge the final patches and not some intermediate version. Also, if looking good, I might just push directly to the repository.

File: package/kernel/broadcom-wl/patches/917-fix-compilation-for-5_10.patch [hunk 1]
```diff
@@ -0,0 +1,42 @@
+Index: broadcom-wl-5.10.56.27.3/driver/wl_linux.c
+===================================================================
+--- broadcom-wl-5.10.56.27.3.orig/driver/wl_linux.c
++++ broadcom-wl-5.10.56.27.3/driver/wl_linux.c
```
Comment (hauke) [inline package/kernel/broadcom-wl/patches/917-fix-compilation-for-5_10.patch:4]: Please run:
```
make package/broadcom-wl/{clean,refresh}
```

**Comment by hauke** [comment]
Thanks for adding support to this upstream. Are you using the broadcom-wl version from OpenWrt?

**Comment by ffainelli** [comment]
> Thanks for adding support to this upstream. Are you using the broadcom-wl version from OpenWrt?

Unfortunately it does not work, wlc_attach fails with error 13, if you happen to know what that means?

And b43 currently fails with:
```
[   31.139608] b43-phy0: Broadcom 5357 WLAN found (core revision 28)
[   31.146326] b43-phy0: Found PHY: Analog 9, Type 4 (N), Revision 9
[   31.152637] b43-phy0 ERROR: FOUND UNSUPPORTED RADIO (Manuf 0x17F, ID 0x2057, Revision 5, Version 2)
[   31.161911] b43: probe of bcma0:1 failed with error -122
[   31.167660] Broadcom 43xx driver loaded [ Features: NL ]
```

Have not tried brcmsmac yet.

**Comment by ffainelli** [comment]
Pushed into master by taking the patches that got merged upstream.

**Pull request closed without merging**